cnf-certification-test/accesscontrol/resources/resources.go

```diff
@@ -12,12 +12,12 @@ func HasRequestsAndLimitsSet(cut *provider.Container) bool {
 		tnf.ClaimFilePrintf("Container has been found missing resource limits: %s", cut.String())
 		passed = false
 	} else {
-		if cut.Resources.Limits.Cpu().Value() == 0 {
+		if cut.Resources.Limits.Cpu().AsApproximateFloat64() == 0 {
```

Comment:
There is a specific function for comparing quantities to zero in the `resource` package: `IsZero()`. The same comment applies to the rest of the change.

Reply:
Good call!  I do remember seeing this.  I will fix.

Reply:
Why is there `float`? Float comparing is complicating.

Reply:
Well at the time, Value() was only giving an integer, so I changed it to float.  But now I will be changing it to .IsZero() per Juan's recommendation.

Reply:
Take a look on float's problems:
 https://jvns.ca/blog/2023/01/13/examples-of-floating-point-problems/